client/src/app/pages/applications/assessment-actions/components/assessment-actions-table.tsx

```diff
@@ -53,7 +101,7 @@ const AssessmentActionsTable: React.FC<AssessmentActionsTableProps> = ({
           numRenderedColumns={numRenderedColumns}
           noDataEmptyState={
             <div>
-              <NoDataEmptyState title="No assessments are associated with this application." />
+              <NoDataEmptyState title="No Questionnaires are currently available. " />
```

Comment:
I think this empty state is really "No questionnaires are currently available to be taken." right?